SU-HSDO/suhumsci
# SHS-5515: Implementation: Adjustments to vertical cards

# READY FOR REVIEW

## Summary
Adjustments to vertical cards

## Need Review By (Date)
03/22

## Urgency
medium

## Steps to Test
1. Go to this pages
  - https://hs-colorful.suhumsci.loc/collections-cards-continued
  - https://hs-colorful.suhumsci.loc/mixed-pages/mixed-page-raised-cards
  - https://hs-traditional.suhumsci.loc/collections-cards-continued
  - https://hs-traditional.suhumsci.loc/mixed-pages/mixed-page-raised-cards
 2. Confirm vertical cards work and look as expected ([Option C on Figma is when the vertical cards has raised cards as a parent class](https://www.figma.com/file/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?type=design&node-id=737-53&mode=design&t=7zJTdAywLXTFIpp4-0) )on both themes, Colorful and Traditional.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
